enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Summarize.tsx

```diff
@@ -55,22 +52,22 @@ const SummarizeInner = ({
     onClose();
   };
 
-  const hasAggregations = Lib.aggregations(currentQuery, stageIndex).length > 0;
+  const hasAggregations = Lib.aggregations(currentQuery, -1).length > 0;
```

Comment:
nit: create a var `stageIndex` and pass it instead

Reply:
Thanks! Fixed in https://github.com/metabase/metabase/pull/50400/commits/c2c41882b2124dc552dbcc68d10aa4098da292b5